patches/u-boot/2012.10/platform-broadcom-iproc-helix4.patch

```diff
@@ -13860,6 +13927,13 @@ index 0000000..b96e61a
 +	v7_outer_cache_enable();
 +}
 +#endif
++
++extern void v7_iproc_invalidate_dcache_all(void);
++
++void cpu_cache_initialization(void)
```

Comment:
`cpu_cache_initialization()` is declared as a "weak" symbol in arch/arm/cpu/armv7/cpu.c.  This function is called just before booting into Linux from the `cleanup_before_linux()` function.